app/locales/en.json

```diff
@@ -51,6 +61,9 @@
     "launch_notif_header": "Notifications will let you know if you cross paths with an infected person.",
     "launch_notif_subheader": "We won't bother you except to share updates on your potential exposure risks.",
     "launch_notification_access": "Allow notifications",
+    "launch_authority_header": "Healthcare Authorities will give us the local data to know if you cross paths with an infected person.",
+    "launch_authority_subheader": "Automatically subscribe to receive the latest updates from Healthcare Authorities in your area.",
+    "launch_authority_access": "Health authority subscription",
```

Comment:
This is my main issue, it's ambiguous on the final screen:

<img width="309" alt="Screen Shot 2020-04-24 at 4 16 06 PM" src="https://user-images.githubusercontent.com/702990/80264877-7cef9780-864a-11ea-8dbf-1591d9c6e3c9.png">

Reply:
I like you're proposed update: `Subscribe to nearby authorities`. I think that along with the longer subtext in that screen _should_ be enough context for a user. Maybe this is something, in particular, we can ask the testers to give us thoughts on?

Reply:
Diarmid could probably weigh in ahead of time too